projects/anti_scaling/distillation.py

```diff
@@ -871,8 +895,13 @@ def compute_loss(self, batch, return_output=False):
 
         fwd_pass = self._perform_forward_passes(batch)
 
+        if hasattr(self.model, 'module'):
```

Comment:
Leave a comment please, even though this is common enough

Also maybe just roll onto one line:
`student_model = self.model.module if hasattr(self.model, 'module') else self.model`

Reply:
Definitely - added a comment. Just checked, and that code doesn't fit on one line